### PR TITLE
Feature/code convention

### PR DESCRIPTION
**Pull request by Yaaappee**
<a href='#crh-start'></a><a href='#crh-data-%7B%22processed%22%3A%20%5B%22https%3A//github.com/ITsvetkoFF/Kv-013/pull/6%23discussion_r56378914%22%2C%20%22https%3A//github.com/ITsvetkoFF/Kv-013/pull/6%23discussion_r56379326%22%2C%20%22https%3A//github.com/ITsvetkoFF/Kv-013/pull/6%23discussion_r56380932%22%2C%20%22https%3A//github.com/ITsvetkoFF/Kv-013/pull/6%23discussion_r56381042%22%2C%20%22https%3A//github.com/ITsvetkoFF/Kv-013/pull/6%23discussion_r56381223%22%2C%20%22https%3A//github.com/ITsvetkoFF/Kv-013/pull/6%23discussion_r56381676%22%2C%20%22https%3A//github.com/ITsvetkoFF/Kv-013/pull/6%23issuecomment-198394149%22%5D%2C%20%22comments%22%3A%20%7B%22General%20Comment%22%3A%20%7B%22html_url%22%3A%20%22https%3A//github.com/ITsvetkoFF/Kv-013/pull/6%23issuecomment-198394149%22%2C%20%22comments%22%3A%20%5B%7B%22body%22%3A%20%22%5Cu0434%5Cu043b%5Cu0438%5Cu043d%5Cu043d%5Cu0430%20%5Cu043c%5Cu0435%5Cu0442%5Cu043e%5Cu0434%5Cu0430%20%5Cu043d%5Cu0435%20%5Cu0431%5Cu043e%5Cu043b%5Cu0435%5Cu0435%2050%20%5Cu0441%5Cu0442%5Cu0440%5Cu043e%5Cu043a%21%22%2C%20%22created_at%22%3A%20%222016-03-18T14%3A49%3A46Z%22%2C%20%22user%22%3A%20%7B%22avatar_url%22%3A%20%22https%3A//avatars.githubusercontent.com/u/7273003%3Fv%3D3%22%2C%20%22html_url%22%3A%20%22https%3A//github.com/ITsvetkoFF%22%7D%7D%5D%2C%20%22title%22%3A%20%22General%20Comment%22%7D%2C%20%22Pull%208dfb0ecdc645417d3eea1afda00fe31ef49be7b8%20DevelopGitHub/GitHubExtension/GitHubExtension.Tests/packages.config%203%22%3A%20%7B%22html_url%22%3A%20%22https%3A//github.com/ITsvetkoFF/Kv-013/pull/6%23discussion_r56381223%22%2C%20%22comments%22%3A%20%5B%7B%22body%22%3A%20%22Can%20you%20explain%20what%20the%20sense%20of%20then%20package%20editconfig%3F%22%2C%20%22created_at%22%3A%20%222016-03-16T17%3A41%3A45Z%22%2C%20%22user%22%3A%20%7B%22avatar_url%22%3A%20%22https%3A//avatars.githubusercontent.com/u/6068068%3Fv%3D3%22%2C%20%22html_url%22%3A%20%22https%3A//github.com/TerraVenil%22%7D%7D%5D%2C%20%22title%22%3A%20%22File%3A%20DevelopGitHub/GitHubExtension/GitHubExtension.Tests/packages.config%3AL1-15%22%7D%2C%20%22Pull%208dfb0ecdc645417d3eea1afda00fe31ef49be7b8%20DevelopGitHub/GitHubExtension/GitHubExtension.Entities/Class1.cs%2010%22%3A%20%7B%22html_url%22%3A%20%22https%3A//github.com/ITsvetkoFF/Kv-013/pull/6%23discussion_r56379326%22%2C%20%22comments%22%3A%20%5B%7B%22body%22%3A%20%22Can%20you%20explain%20what%20the%20purpose%20commit%20Class1.cs%20file%3F%22%2C%20%22created_at%22%3A%20%222016-03-16T17%3A31%3A27Z%22%2C%20%22user%22%3A%20%7B%22avatar_url%22%3A%20%22https%3A//avatars.githubusercontent.com/u/6068068%3Fv%3D3%22%2C%20%22html_url%22%3A%20%22https%3A//github.com/TerraVenil%22%7D%7D%5D%2C%20%22title%22%3A%20%22File%3A%20DevelopGitHub/GitHubExtension/GitHubExtension.Entities/Class1.cs%3AL1-14%22%7D%2C%20%22Pull%208dfb0ecdc645417d3eea1afda00fe31ef49be7b8%20DevelopGitHub/GitHubExtension/GitHubExtension.Tests/Class1.cs%2010%22%3A%20%7B%22html_url%22%3A%20%22https%3A//github.com/ITsvetkoFF/Kv-013/pull/6%23discussion_r56380932%22%2C%20%22comments%22%3A%20%5B%7B%22body%22%3A%20%22The%20same.%22%2C%20%22created_at%22%3A%20%222016-03-16T17%3A39%3A57Z%22%2C%20%22user%22%3A%20%7B%22avatar_url%22%3A%20%22https%3A//avatars.githubusercontent.com/u/6068068%3Fv%3D3%22%2C%20%22html_url%22%3A%20%22https%3A//github.com/TerraVenil%22%7D%7D%5D%2C%20%22title%22%3A%20%22File%3A%20DevelopGitHub/GitHubExtension/GitHubExtension.Tests/Class1.cs%3AL1-14%22%7D%2C%20%22Pull%208dfb0ecdc645417d3eea1afda00fe31ef49be7b8%20DevelopGitHub/GitHubExtension/.editorconfig%201%22%3A%20%7B%22html_url%22%3A%20%22https%3A//github.com/ITsvetkoFF/Kv-013/pull/6%23discussion_r56381676%22%2C%20%22comments%22%3A%20%5B%7B%22body%22%3A%20%22Can%20you%20explain%20why%20we%20need%20this%20package%20in%20the%20project%3F%22%2C%20%22created_at%22%3A%20%222016-03-16T17%3A44%3A17Z%22%2C%20%22user%22%3A%20%7B%22avatar_url%22%3A%20%22https%3A//avatars.githubusercontent.com/u/6068068%3Fv%3D3%22%2C%20%22html_url%22%3A%20%22https%3A//github.com/TerraVenil%22%7D%7D%5D%2C%20%22title%22%3A%20%22File%3A%20DevelopGitHub/GitHubExtension/.editorconfig%3AL1-26%22%7D%2C%20%22Pull%208dfb0ecdc645417d3eea1afda00fe31ef49be7b8%20DevelopGitHub/GitHubExtension/GitHubExtension.Domain/Class1.cs%209%22%3A%20%7B%22html_url%22%3A%20%22https%3A//github.com/ITsvetkoFF/Kv-013/pull/6%23discussion_r56378914%22%2C%20%22comments%22%3A%20%5B%7B%22body%22%3A%20%22Can%20you%20explain%20what%20the%20purpose%20commit%20Class1.cs%20file%3F%22%2C%20%22created_at%22%3A%20%222016-03-16T17%3A29%3A24Z%22%2C%20%22user%22%3A%20%7B%22avatar_url%22%3A%20%22https%3A//avatars.githubusercontent.com/u/6068068%3Fv%3D3%22%2C%20%22html_url%22%3A%20%22https%3A//github.com/TerraVenil%22%7D%7D%5D%2C%20%22title%22%3A%20%22File%3A%20DevelopGitHub/GitHubExtension/GitHubExtension.Domain/Class1.cs%3AL1-13%22%7D%2C%20%22Pull%208dfb0ecdc645417d3eea1afda00fe31ef49be7b8%20DevelopGitHub/GitHubExtension/GitHubExtension.Tests/Settings.StyleCop%201%22%3A%20%7B%22html_url%22%3A%20%22https%3A//github.com/ITsvetkoFF/Kv-013/pull/6%23discussion_r56381042%22%2C%20%22comments%22%3A%20%5B%7B%22body%22%3A%20%22What%20do%20you%20thinks%20is%20it%20optimal%20to%20have%20StyleCop%20settings%20file%20inside%20each%20project%3F%22%2C%20%22created_at%22%3A%20%222016-03-16T17%3A40%3A40Z%22%2C%20%22user%22%3A%20%7B%22avatar_url%22%3A%20%22https%3A//avatars.githubusercontent.com/u/6068068%3Fv%3D3%22%2C%20%22html_url%22%3A%20%22https%3A//github.com/TerraVenil%22%7D%7D%5D%2C%20%22title%22%3A%20%22File%3A%20DevelopGitHub/GitHubExtension/GitHubExtension.Tests/Settings.StyleCop%3AL1-164%22%7D%7D%7D'></a>
<a href='https://www.codereviewhub.com/'></a>
- [ ] <a href='#crh-comment-Pull 8dfb0ecdc645417d3eea1afda00fe31ef49be7b8 DevelopGitHub/GitHubExtension/GitHubExtension.Domain/Class1.cs 9'></a> &nbsp;<b><a href='https://github.com/ITsvetkoFF/Kv-013/pull/6#discussion_r56378914'>File: DevelopGitHub/GitHubExtension/GitHubExtension.Domain/Class1.cs:L1-13</a></b>
- <a href='https://github.com/TerraVenil'></a> Can you explain what the purpose commit Class1.cs file?
- [ ] <a href='#crh-comment-Pull 8dfb0ecdc645417d3eea1afda00fe31ef49be7b8 DevelopGitHub/GitHubExtension/GitHubExtension.Entities/Class1.cs 10'></a> &nbsp;<b><a href='https://github.com/ITsvetkoFF/Kv-013/pull/6#discussion_r56379326'>File: DevelopGitHub/GitHubExtension/GitHubExtension.Entities/Class1.cs:L1-14</a></b>
- <a href='https://github.com/TerraVenil'></a> Can you explain what the purpose commit Class1.cs file?
- [ ] <a href='#crh-comment-Pull 8dfb0ecdc645417d3eea1afda00fe31ef49be7b8 DevelopGitHub/GitHubExtension/GitHubExtension.Tests/Class1.cs 10'></a> &nbsp;<b><a href='https://github.com/ITsvetkoFF/Kv-013/pull/6#discussion_r56380932'>File: DevelopGitHub/GitHubExtension/GitHubExtension.Tests/Class1.cs:L1-14</a></b>
- <a href='https://github.com/TerraVenil'></a> The same.
- [ ] <a href='#crh-comment-Pull 8dfb0ecdc645417d3eea1afda00fe31ef49be7b8 DevelopGitHub/GitHubExtension/GitHubExtension.Tests/Settings.StyleCop 1'></a> &nbsp;<b><a href='https://github.com/ITsvetkoFF/Kv-013/pull/6#discussion_r56381042'>File: DevelopGitHub/GitHubExtension/GitHubExtension.Tests/Settings.StyleCop:L1-164</a></b>
- <a href='https://github.com/TerraVenil'></a> What do you thinks is it optimal to have StyleCop settings file inside each project?
- [ ] <a href='#crh-comment-Pull 8dfb0ecdc645417d3eea1afda00fe31ef49be7b8 DevelopGitHub/GitHubExtension/GitHubExtension.Tests/packages.config 3'></a> &nbsp;<b><a href='https://github.com/ITsvetkoFF/Kv-013/pull/6#discussion_r56381223'>File: DevelopGitHub/GitHubExtension/GitHubExtension.Tests/packages.config:L1-15</a></b>
- <a href='https://github.com/TerraVenil'></a> Can you explain what the sense of then package editconfig?
- [ ] <a href='#crh-comment-Pull 8dfb0ecdc645417d3eea1afda00fe31ef49be7b8 DevelopGitHub/GitHubExtension/.editorconfig 1'></a> &nbsp;<b><a href='https://github.com/ITsvetkoFF/Kv-013/pull/6#discussion_r56381676'>File: DevelopGitHub/GitHubExtension/.editorconfig:L1-26</a></b>
- <a href='https://github.com/TerraVenil'></a> Can you explain why we need this package in the project?
- [ ] <a href='#crh-comment-General Comment'></a> &nbsp;<b><a href='https://github.com/ITsvetkoFF/Kv-013/pull/6#issuecomment-198394149'>General Comment</a></b>
- <a href='https://github.com/ITsvetkoFF'></a> длинна метода не более 50 строк!

<a href='https://www.codereviewhub.com/ITsvetkoFF/Kv-013/pull/6?mark_as_completed=1'></a>&nbsp;<a href='https://www.codereviewhub.com/ITsvetkoFF/Kv-013/pull/6?approve=1'></a>&nbsp;<a href='https://github.com/ITsvetkoFF/Kv-013/pull/6'></a>
<a href='#crh-end'></a>

### CODE REVIEW

File: DevelopGitHub/GitHubExtension/GitHubExtension.Domain/Class1.cs [hunk 1]
```diff
@@ -0,0 +1,12 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace GitHubExtension.Domain
+{
+    public class Class1
```
Comment (TerraVenil) [inline DevelopGitHub/GitHubExtension/GitHubExtension.Domain/Class1.cs]: Can you explain what the purpose commit Class1.cs file?

File: DevelopGitHub/GitHubExtension/GitHubExtension.Entities/Class1.cs [hunk 2]
```diff
@@ -0,0 +1,13 @@
+﻿
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace GitHubExtension.Entities
+{
+    public class Class1
```
Comment (TerraVenil) [inline DevelopGitHub/GitHubExtension/GitHubExtension.Entities/Class1.cs]: The same.

Comment (TerraVenil) [inline DevelopGitHub/GitHubExtension/GitHubExtension.Entities/Class1.cs]: Can you explain what the purpose commit Class1.cs file?

File: DevelopGitHub/GitHubExtension/GitHubExtension.Tests/Class1.cs [hunk 3]
```diff
@@ -0,0 +1,13 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using Xunit;
+
+namespace GitHubExtension.Tests
+{
+    public class Class1
```
Comment (TerraVenil) [inline DevelopGitHub/GitHubExtension/GitHubExtension.Tests/Class1.cs]: The same.

File: DevelopGitHub/GitHubExtension/GitHubExtension.Tests/Settings.StyleCop [hunk 4]
```diff
@@ -0,0 +1,163 @@
+<StyleCopSettings Version="105">
```
Comment (TerraVenil) [inline DevelopGitHub/GitHubExtension/GitHubExtension.Tests/Settings.StyleCop]: What do you thinks is it optimal to have StyleCop settings file inside each project?

File: DevelopGitHub/GitHubExtension/GitHubExtension.Tests/packages.config [hunk 5]
```diff
@@ -0,0 +1,14 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
+<packages>
+  <package id="editorconfig" version="0.12.1" targetFramework="net45" />
```
Comment (TerraVenil) [inline DevelopGitHub/GitHubExtension/GitHubExtension.Tests/packages.config]: Can you explain what the sense of then package editconfig?

File: DevelopGitHub/GitHubExtension/.editorconfig [hunk 6]
```diff
@@ -0,0 +1,25 @@
+# EditorConfig is awesome: http://EditorConfig.org
```
Comment (TerraVenil) [inline DevelopGitHub/GitHubExtension/.editorconfig]: Can you explain why we need this package in the project?

**Comment by ITsvetkoFF** [comment]
длинна метода не более 50 строк!

**Pull request closed without merging**